Editor/Window/RegisterComputeInput.cs

```diff
@@ -23,13 +23,18 @@ public class RegisterComputeInput : StatefulInput
         private readonly GameLiftComputeManager _computeManager;
         private readonly ConnectToFleetInput _fleetDetails;
         private readonly StateManager _stateManager;
+        private StatusBox _registerComputeStatusBox;
+        
+        private const string URLText = "View Logs";
+        private const string ErrorText = "An error occurred when trying to register a compute: ";
```

Comment:
The strings being used should come from the Strings factory (which are enums/constants), and not be hardcoded strings.

Reply:
Fixed